### DISCUSSION TITLE
2주차 논의

### DISCUSSION THREAD

**Discussion by eeeesong** [d-h-k/issue-tracker#52] [Ideas]
### 토큰 만료 확인

- 스타브와 논의 (API?)

  

### 로그아웃

- 서버
  - 유저가 로그아웃 신청할 경우 어떻게 처리? 
  - 관련 지식 필요
- 클라이언트
  - 버튼 클릭 시 로그인 페이지로 이동
  - Keychain 정보 삭제

### 깃허브 사진 탭바 연동

- [참고 링크](https://iosrevisited.blogspot.com/2019/08/add-imageview-profile-picture-to-tabbar-swift-ios.html)
- 잭슨이 한번 해보시기로!

## 리팩토링

- 정리 후 머지, 다음 진행
- 6/14 15시 완료 -> `머지`!

#### PR

- 토큰 만료 부분 구현 후 보내는 걸로
- UI 구현은 동시에 진행하되 브랜치 분리 -> OAuth 부분까지만 1차 PR 

## 레이블 / 마일스톤 UI
- 테이블 뷰 활용 (중간 라인 필요)
- 롤로 담당!

**Comment by eeeesong** [comment]
https://hwannny.tistory.com/71

**Comment by JacksonPk** [comment]
현재는 Jwt 토큰 2일만료로 수정.
자동업데이트는 안된다. 
클라단에서 jwt를 확인해보고 만료됐으면 다시 요청을 해야한다.
REST API에 무조건 헤더값에 jwt가 필요하다.

만약 로그아웃시 jwt를 소멸시키는 api 요청도 되는것인가?

일단은 jwt 만료 체크를 우선으로
